python/orca/src/bigdl/orca/common.py

```diff
@@ -167,12 +167,14 @@ def _check_python_micro_version():
                            f"with micro version >= 10 (e.g. 3.{sys.version_info[1]}.10)")
 
 
-def init_orca_context(cluster_mode=None, cores=2, memory="2g", num_nodes=1,
+def init_orca_context(runtime="spark", cluster_mode=None, cores=2, memory="2g", num_nodes=1,
                       init_ray_on_spark=False, **kwargs):
     """
     Creates or gets a SparkContext for different Spark cluster modes (and launch Ray services
     across the cluster if necessary).
 
+    :param runtime: The runtime for backend. One of "ray", "ray_on_spark" and "spark",
```

Comment:
Change to `"spark" and "ray"`

In future, we may want to support "spark_and_ray"